packages copy/1xdev-users/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@1xdev/users",
```

Comment:
The new package should have a different name than the existing packages, otherwise it will create a conflict.